### ISSUE TITLE
Training scripts

### ISSUE THREAD

**Issue by zoharbarzelay** [liren2515/DIG#2]
Super cool work, thanks for sharing the code!
Would you plz share the training scripts?
Thx!

**Comment by liren2515** [comment]
Thanks for the interesting in our work. 

You can find the necessary information for training our model from [here](https://github.com/liren2515/DIG/tree/main/models). In model_sdf.py and model_skinning.py, you can find how the loss terms are implemented and what values we use for weight balancing. We mainly call function `optimize_parameters` for the training.
